README.md

```diff
@@ -279,6 +282,18 @@ system.
 > depots are ignored. We use `JULIA_DEPOT_PATH` extensively in `slurm-buildkite`.
 
 
+## Frequently Asked Questions
+
+### My Caltech Box link looks different
+
+Your Box link should look something like
+```
+https://caltech.box.com/shared/static/pdsre5tumpc04qbomzjduw07ryd3emwj.gz
+```
+It has to have a `/shared/static` portion and an extension. If that's not the
+case, chances are you copied the link from the wrong page. Make sure you are in
+the "Link Settings" page (see screenshot above).
+
```

Comment:
When this didn't work for me I got a `LoadError` preceded by the following: 
```bash
ERROR: /var/folders/4d/XXX/XX/....
Can not open the file as archive
```
Guessing first time users would not recognize that the error is their box link? I would say the easy check is to `cd` to the artifact directory `~/.julia/artifacts/XXX` and see if the object has actually been downloaded.  Suggest to revise section header as follows (or include in the first sentence or two):
### `ClimaArtifactsHelper` is not able to load my artifact
and then point to checking the caltech box link...